docs/CompatibleSoftware.rst

```diff
@@ -89,35 +281,26 @@ also provided as python code in the config as well).
 
 A `video demonstration <http://vimeo.com/38773736>`__ of the Nuke OCIO workflow.
 
-Mari
-****
-
-`3D Paint - The Foundry <http://www.thefoundry.co.uk/products/mari>`__
-
-Mari 1.4v1+ ships with native support for OpenColorIO in their display toolbar.
-
-A `video demonstration <http://vimeo.com/32909648>`__ of the Mari OCIO workflow.
-
-Katana
-******
 
-`CG Pipeline / Lighting Tool - The Foundry <http://www.thefoundry.co.uk/products/katana>`__
+OpenImageIO
+***********
 
-Color management in Katana (all versions) natively relies on OCIO.
+`Open Source Image Library / Renderer Texture Engine <http://openimageio.org>`__
 
-2D Nodes: OCIODisplay, OCIOColorSpace, OCIOCDLTransform
-Monitor Panel: Full OCIO Support
+Available in the current code trunk. Integration is with makecolortx (allowing
```

Comment:
The OpenImageIO section is outdated and wrong. How about replacing the entire description with:

OIIO's C++ and Python bindings include several methods for applying color transforms to whole images, notably functions in the `ImageBufAlgo` namespace including `colorconvert()`, `ociolook()`, `ociodisplay()`, `ociofiletransform()`. These are also available as part of the `oiiotool` command line utility (`--colorconvert`, `--ociolook`, `--ociodisplay`, `--ociofiletransform`) and the `maketx` utility for preparing textures also supports `--colorconvert`. From C++, there is additional low-level functionality in the header `OpenImageIO/color.h` that are wrappers for accessing underlying OCIO color configurations and doing color processing on individual pixel values.